manifests/addons/dashboards/istio-performance-dashboard.json

```diff
@@ -822,10 +765,10 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(container_fs_usage_bytes{ container=\"istio-proxy\"})",
+          "expr": "sum(container_fs_usage_bytes{container=\"istio-proxy\"})by(name)",
           "format": "time_series",
           "intervalFactor": 2,
-          "legendFormat": "{{ container }}",
+          "legendFormat": "{{ name }}",
```

Comment:
Here I aggregate those metrics data by `name`, also updated the legendFormat to be `name` since we only have one container `istio-proxy`. If by `container`, it would be the total? or `total` is what we want?

If by `name` we will get after I deployed bookinfo app:

<img width="1288" alt="Screen Shot 2020-10-13 at 12 51 40 AM" src="https://user-images.githubusercontent.com/29208297/95915440-f778b480-0d5b-11eb-806c-6db2028ddb58.png">

Reply:
I think we want this to be aggregated though - we can can the label to something else though. Not sure what the normal practice is with only one series for grafana, if we just make it "Total" or what?

Reply:
This *should not be* grouped by name.

Reply:
for example `Total (k8s)` is useful here.

Reply:
yes, both memory and disk legendFormat should be changed to `Total (k8s)`